Blockzilla/SearchSuggestClient.swift

```diff
@@ -17,6 +17,11 @@ class SearchSuggestClient {
     }
     
     func getSuggestions(_ query: String, callback: @escaping (_ response: [String]?, _ error: NSError?) -> Void) {
+        if AppInfo.isTesting() {
```

Comment:
In the interest of not cluttering the actual codebase with testing checks, and since these are all returning dummy values anyway, would it be preferable to just use the network call? I know this increases the surface area of the test (since we're now relying on a network connection), but nearly all of our tests already require that.